wagn/lib/wagn/commands.rb

```diff
@@ -36,19 +36,27 @@ def find_spec_file filename, base_dir
   end
 end
 
+LOAD_TASKS= {
+  'reseed' => 'create',
+  'seed' => 'recreate',
+  'load' => 'seed'
+}
 
 if supported_rails_command? ARGV.first
   if ARGV.delete('--rescue')
     ENV["PRY_RESCUE_RAILS"]="1"
   end
-  require 'wagn'
+  command = ARGV.first
+  command = ALIAS[command] || command
+  require 'generators/card' if command == 'generate' # without this, the card generators don't list with: wagn g --help
```

Comment:
I think what we actually need is to add the generators in an initializer, probably on the engine.